exporters/stats/stackdriver/src/main/java/io/opencensus/exporter/stats/stackdriver/Util.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.exporter.stats.stackdriver;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Strings;
+
+final class Util {
```

Comment:
These util methods can be under `StackdriverExportUtils`.

Reply:
done.